cephfs/makedirs_test.go

```diff
@@ -0,0 +1,30 @@
+//go:build ceph_preview
+// +build ceph_preview
+
+package cephfs
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMakeDirs(t *testing.T) {
+	mount := fsConnect(t)
+	defer fsDisconnect(t, mount)
+
+	dir1 := "/base/sub/way"
+	err := mount.MakeDirs(dir1, 0755)
+	assert.NoError(t, err)
+	defer func() {
+		assert.NoError(t, mount.RemoveDir("/base/sub/way"))
+		assert.NoError(t, mount.RemoveDir("/base/sub"))
+		assert.NoError(t, mount.RemoveDir("/base"))
+	}()
+
+	dir, err := mount.OpenDir(dir1)
+	assert.NoError(t, err)
+	assert.NotNil(t, dir)
+	err = dir.Close()
+	assert.NoError(t, err)
```

Comment:
It's just a matter of personal choice and I don't insist. But how about including the `dir.Close()` inside `assert.NoError` as below:
```
assert.NoError(t, dir.Close())
```